source/code/providers/Container_ContainerInventory_Class_Provider.cpp

```diff
@@ -312,14 +312,7 @@ class ContainerQuery
         openlog("Container_ContainerInventory", LOG_PID | LOG_NDELAY, LOG_LOCAL1);
 
         // Get computer name
-        char name[256];
-        string hostname = gethostname(name, 256) ? "Unknown" : string(name);
-        // in case get full name, extract up to '.'
-        size_t dotpos = hostname.find('.');
-        if (dotpos != string::npos)
-        {
-            hostname = hostname.substr(0, dotpos);
-        }
+        string hostname = getDockerHostName();
```

Comment:
As mentioned above: Really needed? getDockerHostName is a lot more expensive than a local gethostname() call ...